pilot/pkg/serviceregistry/mcp/controller.go

```diff
@@ -76,10 +76,6 @@ func NewController(options *Options) Controller {
 	descriptorsByMessageName := make(map[string]schema.Instance, len(schemas.Istio))
 	synced := make(map[string]bool)
 	for _, descriptor := range schemas.Istio {
-		// do not register SSE as there is a dedicated controller
-		if descriptor.Collection == schemas.SyntheticServiceEntry.Collection {
```

Comment:
why change this?

Reply:
schemas.Istio does not include SyntheticServiceEntry anymore.